addons/website_sale_delivery/models/sale_order.py

```diff
@@ -11,6 +11,7 @@ class SaleOrder(models.Model):
         compute='_compute_amount_delivery',
         string='Delivery Amount',
         help="The amount without tax.", store=True, tracking=True)
+    access_point_address = fields.Json('Delivery Point Address')
```

Comment:
Ho, please don't!

There is no point in this field being `Json`.
It's only in JSON because the FedEx API expects JSON. What if the next delivery carrier wants it in XML?
This field should be a many2one to a partner and converted to JSON in the `website_delivery_fedex` module.

Reply:
@KangOl That's not the point. The user could change the relay point multiple times on the website and we don't want to fill the database with useless partner each time he switch. Anyway the carrier result format can be handle by the function _*delivery_name*_get_close_locations and use a generic js that handle a list of dictionary containing the pickup points data. We could not use this field and create a partner each time he select a pickup point but maintaining the pickup data in a json before the validation seems a better idea for us.

Reply:
and it's also used by sendcloud

Reply:
@KangOl This is not right. The usage of a partner instead of a text/json field would be incorrect as we use more than just the address saved in the partner 

https://github.com/odoo/enterprise/pull/33986/files#diff-12e2d4e0acd2657ca73a282e9d921fc35f10e7787a68623ff1e56367ab86fa52:~:text=.get_fedex_access_point_address()-,fedex_location%20%3D%20order.access_point_address%20if%20order.access_point_address%20else%20False,hold_at_loc.LocationId%20%3D%20fedex_loc_details.get(%27LocationId%27),-4  

And I agree with @amoyaux on the usage of a json instead of a text field.